package.json

```diff
@@ -35,6 +35,7 @@
   "dependencies": {
     "@web-stories-wp/activation-notice": "*",
     "@web-stories-wp/animation": "*",
```

Comment:
Do we need these need to be top level now? CC @swissspidy ?

Reply:
That's a great question. The npm workspace doc has no such requirement. We should actually be able to make the entire `dependencies` empty. 

https://docs.npmjs.com/cli/v7/using-npm/workspaces

Reply:
Probably not completely empty because of storybook and other trst tools?

let‘s give it a try!

Reply:
Correct, since they aren't being bundled, they should be moved to`devDependencies`. Done here https://github.com/google/web-stories-wp/pull/8646/commits/1677f4e6c5d00114e6c9179075b6ed4926d4d4d0